src/components/GridLayout.js

```diff
@@ -298,7 +298,11 @@ class GridLayout extends Component {
               title={shortTitle}
               onClick={() => history.push(`/full/${tile.id}`)}
             >
-              <Map isFull={false} bbox={tile.bbox} tile={tile} />
+              {tile.imageUrl && tile.renderAsImage ? (
```

Comment:
Ziet er goed uit.
Misschien is het mogelijk de check '{tile.imageUrl && tile.renderAsImage' slechts een keer te doen als deze check buiten de switch plaatsvind. Of deze switch omzetten naar een if en dit als eerste if statement checken. 
Nu doen we de check 3 keer en dat is misschien niet nodig.

Reply:
Goeie!

Aangepast